code/control/DeployDispatcher.php

```diff
@@ -76,31 +77,54 @@ public function history(SS_HTTPRequest $request) {
 		$start = ($page - 1) * $list->getPageLength();
 		$list->setPageStart((int) $start);
 		if (empty($list)) {
-			return $this->getAPIResponse(['message' => 'No deploy history'], 404);
+			return $this->getAPIResponse(['list' => []], 200);
 		}
 
+		$currentBuild = $this->environment->CurrentBuild();
+
 		foreach ($list as $deployment) {
 			$data[] = [
+				'DeploymentID' => $deployment->ID,
```

Comment:
I just noticed this, probably should have noticed it yesterday.. but DeplyomentID vs deplyomentID vs deployment_id naming standards..? which one are we running with?